kumo-core/pom.xml

```diff
@@ -17,21 +21,9 @@
         </dependency>
 
         <!--logging -->
-        <dependency>
```

Comment:
thanks for cleaning this up! I have meant to update logging on this project forever now. :)